kms/capi/capi.go

```diff
@@ -0,0 +1,715 @@
+//go:build windows
+// +build windows
+
+package capi
+
+import (
+	"bytes"
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"github.com/pkg/errors"
+	"go.step.sm/crypto/kms/apiv1"
+	"go.step.sm/crypto/kms/uri"
+	"go.step.sm/crypto/randutil"
+	"golang.org/x/crypto/cryptobyte"
+	"golang.org/x/crypto/cryptobyte/asn1"
+	"golang.org/x/sys/windows"
+	"io"
+	"math/big"
+	"reflect"
+	"unsafe"
+)
+
+// Scheme is the scheme used in uris.
+const Scheme = "capi"
+
+const (
+	ProviderNameArg  = "provider"
+	ContainerNameArg = "key"
+	HashArg          = "sha1"
+	StoreLocationArg = "store-location" // 'machine', 'user', etc
+	StoreNameArg     = "store"          // 'MY', 'CA', 'ROOT', etc
+	KeyIDArg         = "key-id"
+	SerialNumberArg  = "serial"
+	IssuerNameArg    = "issuer"
+)
+
+var signatureAlgorithmMapping = map[apiv1.SignatureAlgorithm]string{
+	apiv1.UnspecifiedSignAlgorithm: "ECDSA_P256",
+	apiv1.SHA256WithRSA:            "RSA",
+	apiv1.SHA512WithRSA:            "RSA",
+	apiv1.SHA256WithRSAPSS:         "RSA",
+	apiv1.SHA512WithRSAPSS:         "RSA",
```

Comment:
We should remove this, as we're currently not supporting them. 

And you should add `apiv1.SHA384WithRSA`, as we are supporting them. Although for this specific KMS it doesn't matter which we want to use, we should be consistent with the API.